web/themes/joinup/prototype/sass/base/_fontfaces.sass

```diff
@@ -0,0 +1,98 @@
+// Typography
```

Comment:
![google webfonts helper](https://user-images.githubusercontent.com/1923476/27386502-5c971b7a-5696-11e7-9f4b-990b9e1cfa36.png)

There's an option to include files in addition to the `woff` and `woff2`, which are needed in case the [browser wouldn't support them](https://creativemarket.com/blog/the-missing-guide-to-font-formats).

The google apis has a mechanisms that delivers only what is necessary, but since we don't have it, it's necessary to include all types.